### PR TITLE
fix return section top rating

### PR DESCRIPTION
**Pull request by hoangnhutam95**

### CODE REVIEW

**Comment by quangcsp** [comment]
ok

File: .gitignore [hunk 1]
```diff
@@ -13,4 +13,3 @@ Homestead.json
 Homestead.yaml
 .env
 composer.lock
-package-lock.json
```
Comment (at-diephq) [inline .gitignore]: > composer.lock em nhé

File: app/Events/AverageStarBookHandler.php [hunk 2]
```diff
@@ -13,7 +13,7 @@ public function handle($data)
         $this->data = $data;
 
         $currentCountReview = $this->data['book']->reviews->count();
-        $newAverageStar = ($this->data['book']->avg_star * $currentCountReview + $this->data['star']) / ($currentCountReview + 1);
+        $newAverageStar = ($this->data['book']->avg_star * ($currentCountReview - 1) + $this->data['star']) / $currentCountReview;
```
Comment (at-diephq) [inline app/Events/AverageStarBookHandler.php]: 120 kí tự

File: app/Events/AverageStarBookHandler.php [hunk 3]
```diff
@@ -13,7 +13,8 @@ public function handle($data)
         $this->data = $data;
 
         $currentCountReview = $this->data['book']->reviews->count();
-        $newAverageStar = ($this->data['book']->avg_star * $currentCountReview + $this->data['star']) / ($currentCountReview + 1);
+        $newAverageStar = ($this->data['book']->avg_star * ($currentCountReview - 1) + $this->data['star'])
+        / $currentCountReview;
```
Comment (at-diephq) [inline app/Events/AverageStarBookHandler.php]: 4 spaces

**Comment by trinhductoan** [comment]
ok

**Pull request merged**